### ISSUE TITLE
Inconsistencies when changing orientation

### ISSUE THREAD

**Issue by MikErk** [sailfishos/sailfish-office#17]
Hello, 

I am just starting to get some insights into sailfish programming and found a bug relating to landscape/portrait orientation. During pdf viewing when I change the orientation it switches to a different page number. In my case I tested with a 120-page PDF: In landscape mode when I am look at page 2, then in portrait mode his already on page 5/6. The gap increases when I go further down the document. I am guessing that it might be that somehow you only look at the current scroller position when the orientation is changed. I have looked in some source but I wasn't able to find the function where this is handled.

I posted a bug info already on together.jolla: https://together.jolla.com/question/83724/portraitlandscape-mode-inconsistent-in-sailfish-office/

**Comment by anandrkris** [comment]
I experienced this issue today. I was reading a PDF document and navigated to in Appendix *(no bookmarks on PDF sadly :cry: )* in portrait mode and had to change orientation as there was lot of text. And the page focus was gone! I hate to download Android apps so would be grateful for fix soon.
Also, am curious why **Jolla-made** apps are released only with OS updates. Can't we have updates through harbor store?

**Comment by anandrkris** [comment]
@dcaliste - Tagging - Just In case you're interested to fix :smile:

**Comment by sebix** [comment]
Isn't this issue fixed now with #25 merged? Can't observe it with 1.1.7.28

**Comment by dcaliste** [comment]
Yes, it should be OK now, but when testing my patch, I've observed similar strange behaviours that I cannot reproduce later. So hopefully I would say, it's closed, but it's up to the maintainer to decide !
